core/app/models/spree/reimbursement.rb

```diff
@@ -136,6 +136,15 @@ def all_exchanges?
       return_items.all?(&:exchange_processed?)
     end
 
+    # Accepts all return items, saves the reimbursement, and performs the reimbursement
+    #
+    # @api public
+    def return_all!
```

Comment:
Interesting. I also thought we would want the bang-named-variant for anything that could reasonably raise exceptions , not only if that method had a non-dangerous version. In this case, return items unable to be accepted or `IncompleteReimbursementError`